cli/tools/fmt.rs

```diff
@@ -63,8 +61,30 @@ pub async fn format(
   Ok(())
 }
 
+/// Formats markdown (using https://github.com/dprint/dprint-plugin-markdown) and its code blocks
+/// (ts/tsx, js/jsx).
+fn format_markdown(file_text: &str) -> Result<String, String> {
+  let ts_config = get_typescript_config();
+  let md_config = get_markdown_config();
+  dprint_plugin_markdown::format_text(
+    &file_text,
+    &md_config,
+    Box::new(move |tag, text, _| {
+      if matches!(tag, "ts" | "tsx" | "js" | "jsx") {
+        dprint_plugin_typescript::format_text(
+          &PathBuf::from("_rand.".to_owned() + tag),
+          &text,
+          &ts_config,
+        )
+      } else {
+        Ok(text.to_string())
+      }
+    }),
+  )
+}
+
 async fn check_source_files(
-  config: dprint::configuration::Configuration,
+  config: dprint_plugin_typescript::configuration::Configuration,
```

Comment:
Right now only the typescript configuration is passed down here, but not the markdown. Then the `format_markdown` it does `get_typescript_config()`. Perhaps something should be changed?